<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	font-family: Poppins;
	font-size: 16px;
}

QWidget {
	background-color: white;
}

QPushButton {
	border: 1px solid #00007f;
	background-color: white;
	color: #00007f;
	font-size: 14px;
	font-family: Poppins;
	font-weight: 500;
}

QLabel {
	text-align: center;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="continueBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Continuar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	height: 40px;
	border-radius: 20px;
	border: 1px solid #00007f;
	background-color: white;
	color: #00007f;
	font-weight: 500;
	font-size: 16px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="allCheckBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	background-color: transparent;
	height: 21px;
	font-family: Poppins;
	font-size: 16px;
}

QCheckBox::indicator {
    width: 15px; /* Ancho del indicador */
    height: 15px; /* Alto del indicador */
    border-radius: 4px; /* Bordes redondeados */
    border: 2px solid #666; /* Borde del indicador */
}

QCheckBox::indicator:checked {
    background-color: #00007f; /* Color de fondo cuando está marcado */
}

QCheckBox::indicator:unchecked {
    background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string>Todas las muestras</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="meanCheckBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	background-color: transparent;
	height: 21px;
	font-family: Poppins;
	font-size: 16px;
}

QCheckBox::indicator {
    width: 15px; /* Ancho del indicador */
    height: 15px; /* Alto del indicador */
    border-radius: 4px; /* Bordes redondeados */
    border: 2px solid #666; /* Borde del indicador */
}

QCheckBox::indicator:checked {
    background-color: #00007f; /* Color de fondo cuando está marcado */
}

QCheckBox::indicator:unchecked {
    background-color: transparent;
}</string>
    </property>
    <property name="text">
     <string>Promedio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;¿Como desea guardar las muestras?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="allLbl">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Total de muestras: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="meanLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Temp: 123.22, OD: 23.2, pH: 7.2, Turb: 22.2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openGraphBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Ver gráfica</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
